bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/StateDescriptionConverter.java

```diff
@@ -58,15 +58,15 @@ private BigDecimal toBigDecimal(Map<String, String> attributes, String attribute
             try {
                 return new BigDecimal(attrValueText);
             } catch (NumberFormatException nfe) {
-                throw new ConversionException("The attribute '" + attribute
-                        + "' has not a valid decimal number format!", nfe);
+                throw new ConversionException(
+                        "The attribute '" + attribute + "' has not a valid decimal number format!", nfe);
             }
         }
 
         return defaultValue;
     }
 
-    private boolean toBoolean(Map<String, String> attributes, String attribute, boolean defaultValue) {
+    private boolean extractReadOnly(Map<String, String> attributes, String attribute, Boolean defaultValue) {
```

Comment:
Why has this method be renamed from `toBoolean` to `extractReadOnly`?
Reading the implementation of the method, I don't see any specific `readOnly` attribute stuff.
Sure, it is used for the `readOnly` atrribute, but the attribute itself is still a parameter and so not limited to that only use case.